tests/.e2e-env

```diff
@@ -5,7 +5,7 @@ POSTGRES_TABLE="medic"
 POSTGRES_SCHEMA="v1"
 DBT_POSTGRES_USER="postgres"
 DBT_POSTGRES_PASSWORD="postgres"
-DBT_POSTGRES_SCHEMA="dbt"
+DBT_POSTGRES_SCHEMA="v1"
```

Comment:
Is there a reason to change the test schema here?

Reply:
No specific reason other than making it the same as the value for `POSTGRES_SCHEMA`. Happy to change it back if you prefer it to be different.

Reply:
I don't have a preference, but I found it helpful that it matched the [env.template](https://github.com/medic/cht-sync/blob/main/env.template) values. Also, people might believe that the schemas _have to_ be the same to have the tests work, which is not the case.

Reply:
I have reverted it to `dbt`